modules/mobile/app/controllers/mobile/v0/payment_information_controller.rb

```diff
@@ -43,8 +58,35 @@ def ppiu_params
         )
       end
 
+      def lighthouse_ppiu_params
+        params[:routing_number] = params[:financial_institution_routing_number]
+        params.permit(:account_type,
+                      :account_number,
+                      :routing_number)
+      end
+
       def pay_info
-        @pay_info ||= EVSS::PPIU::PaymentAccount.new(ppiu_params)
+        @pay_info ||= if lighthouse?
+                        Lighthouse::DirectDeposit::PaymentAccount.new(lighthouse_ppiu_params)
+                      else
+                        EVSS::PPIU::PaymentAccount.new(evss_ppiu_params)
+                      end
+      end
+
+      def lighthouse_service
+        @client ||= DirectDeposit::Client.new(@current_user.icn)
+      end
+
+      def payment_account
```

Comment:
Is this in use?

Reply:
I meant the payment_account method. I didn't see where it was being called. I may have just missed it.